go.mod

```diff
@@ -156,7 +156,8 @@ require (
 replace (
 	// TODO: fix keyring upstream
 	github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.9-0.20220920132636-63a7c45caa76
+	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23 v0.45.9
```

Comment:
```suggestion
	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
```

Reply:
We point to ics23 [correct](https://github.com/cosmos/cosmos-sdk/tree/v0.45.9/ics23) pythically, no `cosmos-sdk/ics23/go` different from replace methioned in the [link](https://forum.cosmos.network/t/ibc-security-advisory-dragonberry/7702/16).